care/facility/migrations/0489_migrate_daily_rounds.py

```
@@ -0,0 +1,2565 @@
+            constructed_observation_mapping.extend(
+                create_observation_spec(question, responses, parent_id)
+            )
+
+    return constructed_observation_mapping
+
+
+def null_or_empty(value):
+    return value is None or value == ""
+
+
+def migrate_daily_rounds(apps, schema_editor):
```

Ruff (0.8.2)

Too many branches (71 > 12)

(PLR0912)